src/ui/buttons/button.jsx

```diff
@@ -261,14 +261,15 @@ export function Button({
 
   const shouldShowWalletMenu =
     isWallet && instrument && showWalletMenu({ instrument, userIDToken });
+  const borderRadiusClass =
+    borderRadius >= 0 ? CLASS.BORDER_RADIUS : `${CLASS.SHAPE}-${shape}`;
```

Comment:
Nice check here for a valid border radius 💯

Reply:
I actually have to change this to a type check 😅   It throws an error if border radius is undefined.